core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -104,14 +110,28 @@ def configure_application
     end
 
     def install_default_plugins
-      if options[:with_authentication] && (options[:auto_accept] || yes?("
+      if options[:with_authentication] && (options[:auto_accept] || !no?("
   Solidus has a default authentication extension that uses Devise.
-  You can find more info at github.com/solidusio/solidus_auth_devise.
+  You can find more info at https://github.com/solidusio/solidus_auth_devise.
 
   Would you like to install it? (y/n)"))
 
         gem 'solidus_auth_devise'
       end
+
+      payment_method_name = options[:payment_method]
+
+      if !payment_method_name && !options[:auto_accept]
+        available_methods = PAYMENT_METHODS.keys
+
+        payment_method_name = ask("
+  You can now select a payment method to be included in the installation process.
+  Please select a payment method name?", limited_to: available_methods, default: available_methods.first)
```

Comment:
`Please select a payment method name?` -> the question mark in this sentence seems a bit odd to me, what about removing it or changing the wording?